HW-3/index4.js

```diff
@@ -0,0 +1,13 @@
+// Функция принимает строку и проверяет ее на палиндром. В случае если строка является палиндромом возвращается true иначе
+// false.
+
+const isPalindrom = (str) => {
+  const word = str.toLowerCase();
+  if (word === word.split("").reverse().join("")) {
+    return true;
+  }
+  return false;
```

Comment:
every comparison always return bool. no matter to do it like this.
HINT: `return word === word.split("").reverse().join("")`